crates/ide/src/diagnostics.rs

```diff
@@ -253,6 +265,37 @@ mod tests {
         );
     }
 
+    /// Similar to `check_fix`, but applies all the available fixes.
+    fn check_fixes(ra_fixture_before: &str, ra_fixture_after: &str) {
+        let after = trim_indent(ra_fixture_after);
+
+        let (analysis, file_position) = fixture::position(ra_fixture_before);
+        let diagnostic = analysis
+            .diagnostics(&DiagnosticsConfig::default(), file_position.file_id)
+            .unwrap()
+            .pop()
+            .unwrap();
+        let fix = diagnostic.fix.unwrap();
+        let target_file_contents = analysis.file_text(file_position.file_id).unwrap();
+        let actual = {
+            let mut actual = target_file_contents.to_string();
+            // Go from the last one to the first one, so that ranges won't be affected by previous edits.
+            for edit in fix.source_change.source_file_edits.iter().rev() {
+                edit.edit.apply(&mut actual);
+            }
+            actual
+        };
+
+        assert_eq_text!(&after, &actual);
+        assert!(
+            fix.fix_trigger_range.start() <= file_position.offset
+                && fix.fix_trigger_range.end() >= file_position.offset,
```

Comment:
```suggestion
            fix.fix_trigger_range.start() <= file_position.offset
                && file_position.offset <= fix.fix_trigger_range.end(),
```

to give this a better visual intuition

Reply:
Isn't this `TextRange::contains_inclusive`?

Reply:
It was copy-pasted from `check_fix`. Fixed in both places now.